app/_gateway_entities/consumer.md

```diff
@@ -143,7 +145,19 @@ Common use cases for Consumers:
 
 ## Centrally-managed Consumers 
 
-<!-- @todo: migrate from Konnect docs: https://docs.konghq.com/konnect/centralized-consumer-management/ -->
+Consumers can be scoped to a {{site.konnect_short_name}} region and managed centrally, or be scoped to a Control Plane in Gateway Manager.
+
+Centralized Consumer management provides the following benefits:
+* Set up a Consumer identity centrally instead of defining it in multiple Control Planes.
+* Share Consumers across multiple Control Planes. Users don't need to replicate changes to Consumer identity in multiple Control Planes and Consumer configuration doesn't conflict.
+* Reduce configuration sync issues between the Control Plane and the Data Planes. Consumers that are managed centrally aren't part of the configuration that is pushed down from the Control Plane to the Data Planes, so it reduces config size and latency. 
+
+You can manage Consumers centrally using the {{site.konnect_short_name}} API. Only Org Admins and Control Plane Admins have CRUD permissions for these Consumers. When you create a Consumer centrally, you must assign it to a realm. A realm groups Consumes around an identity, defined by organizational boundaries, such as a production realm or a development realm. Realms are connected to a [geographic region](/konnect-geos/) in {{site.konnect_short_name}}. Centrally managed Consumers exist outside of Control Planes, so they can be used across Control Planes.
```

Comment:
```suggestion
Centrally managed Consumers exist outside of Control Planes, so they can be used across Control Planes.

You can manage Consumers centrally using the [{{site.konnect_short_name}} Consumers API](/api/konnect/consumers/v1/). 
Only Org Admins and Control Plane Admins have CRUD permissions for these Consumers. 

When you create a Consumer centrally, you must assign it to a realm. A realm groups Consumers around an identity, defined by organizational boundaries, such as a production realm or a development realm. 
Realms are connected to a [geographic region](/konnect-geos/) in {{site.konnect_short_name}}. 
```
The last sentence feels more like an intro. Also added a link and broke this up a bit.